src/crud.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-plusplus */
+import { savedList } from './interactive.js';
+import {
+  todoTasks, taskInput, btnClear,
+} from './index.js';
+
+export function createTask() {
+  const form = document.querySelector('.to-do');
+
+  form.addEventListener('submit', () => {
+    const newToDo = taskInput.value;
+    if (newToDo) {
+      todoTasks.push({
+        description: newToDo,
+        completed: false,
+        index: todoTasks.length,
+        checked: false,
+      });
+      savedList();
+    }
+  });
+}
```

Comment:
- [ ]  The property `index` has to be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). please fix that!

Reply:
Thank you, I have made the requested modification